test/test.js

```diff
@@ -97,7 +97,7 @@ test('Must parse locals if locals prop is passed and it contains a valid JSON st
 
 test('Must not parse locals if locals prop is passed but is not a valid JSON string', async t => {
   const actual = `<div class="test"><module href="./test/locals.spec.html" locals="test">Test</module></div>`;
-  const expected = `<div class="test"><button type="button">foo is: {{ foo }} - Test</button></div>`;
+  const expected = `<div class="test"><button type="button">foo is: undefined - Test</button></div>`;
```

Comment:
Is this really correct? could this cause regression?

Reply:
I think it's correct, it's what you'd get when using an undefined variable inside an expression. 

I actually think what we had until now was wrong, since you could pass something like `{{ foo }}` inside a module and, although `foo` would be undefined, it would render `{{ foo }}` - also making the use of `@{{ }}` escaping confusing and inconsistent.